Remotion/Web/Development.WebTesting/ScreenshotCreation/Fluent/FluentScreenshotExtensions.cs

```diff
@@ -102,6 +102,17 @@ public static FluentScreenshotElement<T> ScrollIntoView<T> ([NotNull] this Fluen
       return element;
     }
 
+    /// <summary>
+    /// Sets the <see cref="IFluentScreenshotElement.MinimumElementVisibility"/> on the specified <paramref name="element"/>
+    /// to <see cref="ElementVisibility"/>.<see cref="ElementVisibility.PartiallyVisible"/>.
+    /// </summary>
+    public static IFluentScreenshotElement<T> AllowPartialVisibility<T> (this IFluentScreenshotElement<T> element)
```

Comment:
@MichaelKetting This is a feature that is useful but can be solved using `builder.MinimumVisibility = ...` instead